tests/pytorch_pfn_extras_tests/onnx/test_annotate.py

```diff
@@ -142,6 +142,7 @@ def _fn2(x):
 
 
 @pytest.mark.filterwarnings("ignore::torch.jit.TracerWarning")
+@pytest.mark.filterwarnings("ignore::UserWarning")
```

Comment:
Deprecation warning of `torch.floor_divide` raised from here: https://github.com/pytorch/pytorch/blob/v1.9.0/torch/nn/functional.py#L2358